CMakeLists.txt

```diff
@@ -2349,27 +2350,30 @@ if(GPERFTOOLS OR GPERFTOOLSPROFILER)
 endif()
 
 # HSS1394 MIDI device
+#
+# The HSS1394 library is only available on macOS, therefore this option is
+# forcibly set off on all other platforms.
 if(WIN32 OR APPLE)
   find_package(HSS1394)
-  cmake_dependent_option(HSS1394 "HSS1394 MIDI device support" ON "HSS1394_FOUND;WIN32 OR APPLE" OFF)
-  if(HSS1394)
-    target_sources(mixxx-lib PRIVATE
-      src/controllers/midi/hss1394controller.cpp
-      src/controllers/midi/hss1394enumerator.cpp
-    )
-    target_compile_definitions(mixxx-lib PUBLIC __HSS1394__)
-    if(WIN32 OR APPLE)
-      if(NOT HSS1394_FOUND)
-        message(FATAL_ERROR "HSS1394 MIDI device support requires the libhss1394 and its development headers.")
-      endif()
-      target_link_libraries(mixxx-lib PUBLIC HSS1394::HSS1394)
-    endif()
+else()
+  set(HSS1394 OFF)
+endif()
+cmake_dependent_option(HSS1394 "HSS1394 MIDI device support" "${HSS1394_FOUND}" "WIN32 OR APPLE" OFF)
```

Comment:
Why not simply `default_option(... "HSS1394_FOUND")`?

HSS1394_FOUND will only ever be set if WIN32 OR APPLE.

Reply:
Checking the OS first avoids printing an error that it is not found on Linux which could mislead the user into thinking it should be installed.